frontend/src/pages/modelRegistry/screens/RegisterModel/RegisterModel.scss

```diff
@@ -0,0 +1,4 @@
+/* to help with getting the "Autofill" buttons to align properly */
+.pf-v5-c-radio__label {
+    width: 100%;
+}
```

Comment:
We should avoid custom CSS, especially if it touches a PF class across the whole app. Maybe we can change this style rule so it only applies to this specific radio button, or look at an alternative... I'll get back to this in a sec